drop-in/src/main/java/com/adyen/checkout/dropin/internal/ui/GiftCardComponentDialogFragment.kt

```diff
@@ -129,6 +130,12 @@ internal class GiftCardComponentDialogFragment : DropInBottomSheetDialogFragment
         protocol.showError(null, getString(R.string.component_error), componentError.errorMessage, true)
     }
 
+    override fun onCancel(dialog: DialogInterface) {
```

Comment:
I can see that every subclass of `DropInBottomSheetDialogFragment` has this method overridden. It looks like all bottom sheet dialog fragments should have this behavior. What do you think of moving this function to the `DropInBottomSheetDialogFragment` and removing them from all of its subclasses?

`ActionComponentDialogFragment` has a different implementation though. This one could stay.

Reply:
Good point, let me look into it